src/main/scala/is/hail/check/Gen.scala

```diff
@@ -28,6 +28,11 @@ case class Parameters(rng: RandomDataGenerator, size: Int, count: Int) {
 
 object Gen {
 
+  val nonExtremeDouble: Gen[Double] = oneOfGen(
```

Comment:
What is the definition of an extreme double?

Reply:
Something close to infinity, I guess. I'm happy to use something else if there's a more informative name for this.